.husky/pre-commit

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+FILES=$(git diff --cached --name-only --diff-filter=ACMR | sed 's| |\\ |g')
+[ -z "$FILES" ] && exit 0
```

Comment:
Not sure if this is the best way to get staged files. Open to suggestions.

Reply:
What's `--diff-filter=ACMR` ?
I might be wrong but `git diff --cached --name-only` should be enough